core/papersize.lua

```diff
@@ -63,13 +63,16 @@ local papersize = {
 }
 
 setmetatable(papersize, {
-    __call = function (self, size)
+    __call = function (self, size, landscape)
       local _, _, x, y = string.find(size, "(.+)%s+x%s+(.+)")
       if x and y then
         return { SILE.measurement(x):tonumber(), SILE.measurement(y):tonumber() }
       else
         size = string.lower(size:gsub("[-%s]+", ""))
         if self[size] then
+          if SU.boolean(landscape) then
```

Comment:
```suggestion
          if landscape then
```

Lets get this cast to a boolean when we *set* the class option, thereafter just assume it is one.